schema/datatypes/datatypes.fbs

```diff
@@ -136,3 +136,11 @@ struct Temperature {
 struct Ipv4Address  {
     addr: uint32;
 }
+
+table Bytes {
+    b: [uint8];
+}
+
+table String {
+    s: string;
+}
```

Comment:
Get in conflict with the actual String class in java.
The library does not compile